### PR TITLE
Add plugin settings allowing to disable event capturing for certain build configurations/targets

### LINKED ISSUE #262: Ability to turn on & off platforms/configurations from sending crashes to Sentry
What problem could Sentry solve that it doesn't?
While we're currently waiting on #32 support, something that would be useful alongside this would to be able to turn off crashes for certain configurations or platforms, for example, we don't want to send Editor crashes to Sentry.

### PR DESCRIPTION
**Pull request by tustanivsky**
This change should provide extra flexibility for the Sentry initialization by allowing to disable it for certain build configurations/targets via the settings menu:

image

Closes #262

### CODE REVIEW

File: plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp [hunk 1]
```diff
@@ -338,3 +353,63 @@ void USentrySubsystem::DisableAutomaticBreadcrumbs()
 		FCoreDelegates::GameSessionIDChanged.Remove(GameSessionIDChangedDelegate);
 	}
 }
+
+bool USentrySubsystem::IsCurrentBuildConfigurationDisabled()
```
Comment (bitsandfoxes) [inline plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp:357]: Should this be moved behind the self-initialization?

**Review by bitsandfoxes — APPROVED** [review]
🔥

File: plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp [hunk 2]
```diff
@@ -47,13 +48,21 @@ void USentrySubsystem::Deinitialize()
 {
 	DisableAutomaticBreadcrumbs();
 
+	Close();
```
Comment (bitsandfoxes) [inline plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp:51]: Is that an earlier oversight?

Comment (tustanivsky) [reply plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp:51]: Yeah, noticed it while disabling the event capturing for Editor - without shutting down the native SDK upon subsystem's deinitialization events were still passing through in the subsequent game sessions even though the corresponding target was disabled

**Pull request merged**